covert/__main__.py

```diff
@@ -124,28 +127,46 @@ def argparse():
     if a == '--':
       args.files += args
       break
-    if not a.startswith('--') and len(a) > 2:
-      stderr.write(f' 💣  Short arguments cannot be combined: covert {args.mode} {a}\n')
-      sys.exit(1)
     if a.startswith('--'):
       a = a.lower()
-    argvar = next((k for k, v in ad.items() if a in v), None)
-    if argvar is None:
-      stderr.write(f'{modehelp}\n 💣  Unknown argument: covert {args.mode} {a}\n')
-      sys.exit(1)
-    try:
-      var = getattr(args, argvar)
-      if isinstance(var, list):
-        var.append(next(aiter))
-      elif isinstance(var, str):
-        setattr(args, argvar, next(aiter))
-      elif isinstance(var, int):
-        setattr(args, argvar, var + 1)
+    if not a.startswith('--') and len(a) > 2:
+      if a not in longargs:
+        if any(longarg[1:] in a for longarg in longargs) is True:
+          stderr.write(f' 💣  Multiple arguments cannot have long arguments: covert {args.mode} {a}\n')
+          sys.exit(1)
+        elif any(arg not in shortargs for arg in list(a[1:])) == True:
+          falseargs = [arg for arg in list(a[1:]) if arg not in shortargs]
+          stderr.write(f' 💣  {falseargs} is not an argument: covert {args.mode} {a}\n')
+          sys.exit(1)
+        a = [f'-{shortarg}' for shortarg in list(a[1:]) if shortarg in shortargs]
       else:
-        setattr(args, argvar, True)
-    except StopIteration:
-      stderr.write(f'{modehelp}\n 💣  Argument parameter missing: covert {args.mode} {a} …\n')
-      sys.exit(1)
+        a = f'-{a}'
+    if isinstance(a, str) == True:
```

Comment:
I minor thing about Pythonic style. Avoid `== True`, `!= None` and similar comparisons, as you can use a truthiness of something directly on `if` condition or coupled with `not`, like so:

```python
if isinstance(a, str):
```

Sometimes you should use `var is not None` or `var is True` when you need to make sure that var is or isn't *precisely* one of these values and not just any *truthy* value (e.g 1 and True are both truthy but only the latter `is True`). `is` is better with None/True/False, and for checking if two variables refer to the same object, but to compare if two values such as numbers or strings are equal, stick to `==`.